docs/development/sync-protocol.md

```diff
@@ -0,0 +1,431 @@
+---
+title: Santa Sync Server Protocol
+parent: Development
+mermaid: true
+---
+
+{% if page.mermaid %}
+  {% include mermaid.html %}
+{% endif %}
+
+# Overview
+
+This documents describes the protocol between Santa and the Sync Server aka the Sync Protocol. Implementors should be able to use this to create their own Sync Servers.
```

Comment:
Where "sync server" is not in a heading or title, maybe it should be lowercase? There is a bit of a mix throughout. My vote would be for lowercase.

Reply:
I'll make it lower case.